user-interface/src/main/java/life/qbic/datamanager/views/projects/project/rawdata/RawDataDetailsComponent.java

```diff
@@ -76,10 +78,22 @@ public RawDataDetailsComponent(@Autowired RawDataService rawDataService,
    *                   be filtered for
    */
   public void setSearchedRawDataValue(String searchTerm) {
+    if (!this.searchTerm.equals(searchTerm)) {
+      refreshGrids();
+    }
     this.searchTerm = searchTerm;
+  }
+
+  public void refreshGrids() {
+    resetSelectedMeasurements();
     rawDataGridDataViews.forEach(AbstractDataView::refreshAll);
   }
 
+  private void resetSelectedMeasurements() {
+    proteomicsRawDataGrid.clearSelectedItems();
+    ngsRawDataGrid.clearSelectedItems();
+  }
```

Comment:
Consider renaming it to `clearSelectedMeasurements` to match your terminology. Alternatively `clearMeasurementSelection`:wink: